diff -ZNaur aws-sdk-cpp-1.9.149-orig/aws-cpp-sdk-text-to-speech/source/text-to-speech/TextToSpeechManager.cpp aws-sdk-cpp-1.9.149/aws-cpp-sdk-text-to-speech/source/text-to-speech/TextToSpeechManager.cpp
--- aws-sdk-cpp-1.9.149-orig/aws-cpp-sdk-text-to-speech/source/text-to-speech/TextToSpeechManager.cpp	2021-11-20 20:58:56.000000000 -0600
+++ aws-sdk-cpp-1.9.149/aws-cpp-sdk-text-to-speech/source/text-to-speech/TextToSpeechManager.cpp	2021-11-20 22:58:40.326480695 -0600
@@ -146,7 +146,6 @@
                 AWS_LOGSTREAM_TRACE(CLASS_TAG, "Audio retrieved from Polly. " << result.GetContentType() << " with " 
                     << result.GetRequestCharacters() << " characters syntesized");
 
-                std::streamsize amountRead(0);
                 unsigned char buffer[BUFF_SIZE];
 
                 std::lock_guard<std::mutex> m(m_driverLock);
@@ -159,7 +158,6 @@
                     AWS_LOGSTREAM_TRACE(CLASS_TAG, "Writing " << read << " bytes to device.");
 
                     successfullyPlayed = m_activeDriver->WriteBufferToDevice(buffer, (std::size_t)read);
-                    amountRead += read;
                     played = successfullyPlayed;
                 }
 
