--- ./crt/aws-crt-cpp/crt/aws-c-common/cmake/AwsCFlags.cmake.orig	2021-06-25 12:34:07.168145473 -0500
+++ ./crt/aws-crt-cpp/crt/aws-c-common/cmake/AwsCFlags.cmake	2021-06-25 12:34:55.968362249 -0500
@@ -75,7 +75,7 @@
 
        # -moutline-atomics generates code for both older load/store exclusive atomics and also
        # Arm's Large System Extensions (LSE) which scale substantially better on large core count systems
-        check_c_compiler_flag(-moutline-atomics HAS_MOUTLINE_ATOMICS)
+        check_c_compiler_flag("-Werror -moutline-atomics" HAS_MOUTLINE_ATOMICS)
         if (HAS_MOUTLINE_ATOMICS)
             list(APPEND AWS_C_FLAGS -moutline-atomics)
         endif()
